cmd/atlas/atlas.go

```diff
@@ -94,64 +71,6 @@ func createConfigFromMongoCLIConfig() {
 		f.Close()
 		return
 	}
-
-	p, err := mongoCLIConfigFilePath()
-	if err != nil {
-		return
-	}
-
-	in, err := os.Open(p)
-	if err != nil {
-		return
-	}
-	defer in.Close()
-
-	_, err = os.Stat(atlasConfigHomePath) // check if the dir is already there
-	if err != nil {
-		defaultPermissions := 0700
-		if err = os.Mkdir(atlasConfigHomePath, os.FileMode(defaultPermissions)); err != nil {
-			return
-		}
-	}
```

Comment:
have you tried the changes locally with no folder present? I'm wondering if this is still needed when installing on a clean system

Reply:
can you do do `mv ~/Library/Application\ Support/atlascli ~/Library/Application\ Support/atlasclibck` before doing login? that's the part I'm curios, if the folder doesn't exists

Reply:
```shell
❯ mv ~/Library/Application\ Support/atlascli ~/Library/Application\ Support/atlasclibck
❯ ./bin/atlas login

To verify your account, copy your one-time verification code:
BVMR-QWR3

Paste the code in the browser when prompted to activate your Atlas CLI. Your code will expire after 10 minutes.

To continue, go to https://account.mongodb.com/account/connect
^C%
❯ ./bin/atlas clusters list
Error: this action requires authentication

To log in using your Atlas username and password, run: atlas auth login
To set credentials using API keys, run: atlas config init
❯ git pull
❯ ./bin/atlas config list
PROFILE NAME

```

Reply:
>To continue, go to https://account.mongodb.com/account/connect
^C%

can you continue and see if the file is created? the thing to check if we need to pre create the folder or will it be created along the file, since you are deleting the code that created the folder before trying to create the file

Reply:
yep, worked fine! 

```shell
❯ ./bin/atlas login

To verify your account, copy your one-time verification code:
Q9K3-J896

Paste the code in the browser when prompted to activate your Atlas CLI. Your code will expire after 10 minutes.

To continue, go to https://account.mongodb.com/account/connect
Successfully logged in as blvaguiar@gmail.com.
Select one default organization and one default project.
? Choose a default project: 623202922f697375cac9414a

You have successfully configured your profile.
You can use [atlas config set] to change your profile settings later.
❯ ./bin/atlas config list
PROFILE NAME
default
```

Reply:
thanks for checking this :shipit: